Dockerfile.custom

```
@@ -45,30 +45,18 @@
-# Set Terraform 0.14.x as the default `terraform`. You can still use
-# version 0.15.x by calling `terraform-0.15` or version 1.x as terraform-1
-RUN update-alternatives --set terraform /usr/share/terraform/0.14/bin/terraform
 
 # Pin kubectl minor version (must be within 1 minor version of cluster version)
 # Note, however, that due to Docker layer caching and the structure of this
 # particular Dockerfile, the patch version will not automatically update
 # until you change the minor version or change the base Geodesic version.
 # If you want, you can pin the patch level so you can update it when desired.
-ARG KUBECTL_VERSION=1.20
+ARG KUBECTL_VERSION=1.30
 RUN apt-get update && apt-get install kubectl-${KUBECTL_VERSION}
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)